### PR TITLE
Use pnpm instead of yarn

### PR DESCRIPTION
**Pull request by B2o5T**

### CODE REVIEW

File: scripts/patch-manifests.js [hunk 1]
```diff
@@ -10,7 +10,9 @@ import { createRequire } from 'module';
 
 function patchPackage(name, patchFn) {
   const require = createRequire(import.meta.url);
-  const indexFile = require.resolve(name);
+  const indexFile = require.resolve(name, {
+    paths: [path.join(process.cwd(), 'node_modules', '.pnpm', 'node_modules')],
+  });
```
Comment (B2o5T) [inline scripts/patch-manifests.js:15]: image

File: deployment/utils/remote-artifact-as-service.ts [hunk 2]
```diff
@@ -117,7 +117,7 @@ export class RemoteArtifactAsServiceDeployment {
           volumeMounts,
           command:
             this.options.packageInfo.runtime === 'node'
-              ? ['/bin/sh', '-c', artifactUrl.apply(v => `yarn add ${v}`)]
+              ? ['/bin/sh', '-c', artifactUrl.apply(v => `pnpm add ${v}`)]
```
Comment (kamilkisiela) [inline deployment/utils/remote-artifact-as-service.ts:120]:
```suggestion
              ? ['/bin/sh', '-c', artifactUrl.apply(v => `yarn add ${v}`)]
```

File: deployment/utils/remote-artifact-as-service.ts [hunk 3]
```diff
@@ -151,7 +151,7 @@ export class RemoteArtifactAsServiceDeployment {
             : undefined,
           command:
             this.options.packageInfo.runtime === 'node'
-              ? ['yarn', this.options.bin || this.options.packageInfo.bin]
+              ? ['pnpm', this.options.bin || this.options.packageInfo.bin]
```
Comment (kamilkisiela) [inline deployment/utils/remote-artifact-as-service.ts:154]:
```suggestion
              ? ['yarn', this.options.bin || this.options.packageInfo.bin]
```

File: deployment/utils/remote-artifact-as-service.ts (hunk 2 above)
Comment (enisdenjo) [reply deployment/utils/remote-artifact-as-service.ts:120]: Old code, pnpm is already there.

File: deployment/utils/remote-artifact-as-service.ts (hunk 3 above)
Comment (enisdenjo) [reply deployment/utils/remote-artifact-as-service.ts:154]: Old code, pnpm is already there.

File: deployment/utils/remote-artifact-as-service.ts (hunk 2 above)
Comment (enisdenjo) [reply deployment/utils/remote-artifact-as-service.ts]: https://github.com/kamilkisiela/graphql-hive/pull/477/commits/251d1c4ecf33c43bce896369be09291417f47590

**Review by kamilkisiela — APPROVED** [review]

**Review by dotansimha — APPROVED** [review]

**Comment by dotansimha** [comment]
Staging works great. Merging

**Pull request merged**